manager/utils.go

```diff
@@ -322,6 +326,82 @@ func EnableUprobeEvent(probeType string, funcName, path, UID string, uprobeAttac
 	return uprobeId, nil
 }
 
+func memELFFromProcMaps(pid int, path string) ([]byte, uintptr, error) {
+	p, err := procfs.NewProc(pid)
```

Comment:
Not really the start of a review, but you might want to use `DataDog/gopsutil` here because system-probe might be started from a container, which means that you'd need to query `/host/proc` instead of `/proc`.

Reply:
Not sure `DataDog/gopsutil/process` can export the content of `/proc/[pid]/maps` though ... you might need to make a PR there too 😅